reporters/uniswapSubgraphClient.js

```diff
@@ -26,7 +26,7 @@ function getUniswapClient() {
  * @return query string
  */
 function PAIR_DATA(pairAddress, block) {
-  const blockNumberLag = 3;
+  const blockNumberLag = 5;
```

Comment:
why exactly did you change this?

Reply:
>Finally, I increased the Uniswap subgraph delay parameter to 5 blocks because I noticed an error between the latest block in Infura's mainnet node and the graph. This parameter is used so that if we want to get the current data, for example, we will actually be asking the graph to send us data from 5 (previously 3) blocks behind the latest block according to our web3 node. This is to account for the fact that the graph appears to have a delay in making data available.